prow/entrypoint/run_test.go

```diff
@@ -124,3 +170,101 @@ func compareFileContents(name, file, expected string, t *testing.T) {
 		t.Errorf("%s: expected contents: %q, got %q", name, expected, data)
 	}
 }
+
+func TestGitHeadTimestamp(t *testing.T) {
```

Comment:
This adds a dependency on `git` to the test image environment  -- do we need to update the image for the test job?

Reply:
doesn't seem like it? I think the golang image contains git already, since it's needed for things like `go get`.